### ISSUE TITLE
le-renew.sh should use --deploy-hook

### ISSUE THREAD

**Issue by Feilner** [linuxserver/docker-swag#74]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

------------------------------

## Expected Behavior

I was looking for a deploy hook of swag found #25 and looked at the hooks of certbot.

After calling certbot -n renew the certificate operations should only be executed when the certificate where actually renewed.
The operations should be done by the --deploy-hook of certbot renew

see [Note ](https://certbot.eff.org/docs/using.html)from certbot:
```
certbot renew exit status will only be 1 if a renewal attempt failed. This means certbot renew 
exit status will be 0 if no certificate needs to be updated. If you write a custom script and expect 
to run a command only after a certificate was actually renewed you will need to use the 
--deploy-hook since the exit status will be 0 both on successful renewal and when renewal is 
not necessary.
```

## Current Behavior

Currently certbot is called like this. This means that all operations are executed within the post-hook.
```
  certbot -n renew \
    --post-hook "if ps aux | grep [n]ginx: > /dev/null; then s6-svc -h /var/run/s6/services/nginx; fi; \
    cd /config/keys/letsencrypt && \
    openssl pkcs12 -export -out privkey.pfx -inkey privkey.pem -in cert.pem -certfile chain.pem -passout pass: && \
    sleep 1 && \
    cat privkey.pem fullchain.pem > priv-fullchain-bundle.pem && \
    chown -R abc:abc /config/etc/letsencrypt"
```

## Steps to Reproduce
See above

## Environment
**OS:** Ubuntu Linux
**CPU architecture:** x86_64
**How docker service was installed:**
official docker repo

## Command used to create docker container (run/create/compose/screenshot)

n/a

## Docker logs

n/a

**Comment by aptalca** [comment]
The only difference between a post hook and a deploy hook in this case is that if the renewal fails for some reason, it still restarts nginx and recreates the pfx from the existing cert which is not a big deal.

**Comment by Feilner** [comment]
Just add a friendly ping as I have also pushed a PR that fixes this #89.
